docs: add the first weekly report

Yangxiamao opened the pull request.

Yangxiamao committed:
feat:add 2021-8.9-8.15 weekly report

feat: Add blog comment

lijing-21 reviewed (commented):
- [8 月 21 日 Apache APISIX Meetup 上海站，欢迎报名参加！](https://mp.weixin.qq.com/s/YIoc7XlgZIq0V-CyTDD2VA)  
本次 Meetup 由 API7.AI 发起，并邀请爱奇艺、空中云汇等合作伙伴共同为大家呈现。Apache APISIX PMC 成员、贡献者、社区技术专家汇聚一堂，共同探讨 Apache APISIX 社区发展、行业实践等话题。

- [从新手村出来，我在 Apache APISIX 社区发出了第一个 PR](https://mp.weixin.qq.com/s/pyfBAHONGjkKJhwsjyhdUA)  
屠正松同学的采访文章，下回我们会采访哪位社区的小伙伴呢？快来 Apache APISIX 社区瞧一瞧吧！没准，下一位 committer 就是你！

- [舍弃Kong和Nginx，Apache APISIX 在趣链科技 BaaS 平台的落地实践](https://mp.weixin.qq.com/s/oARDFf_48X99MXBwoXcSHw)  
介绍了 Apache APISIX 在趣链科技 BaaS 平台中的落地实践情况，以及趣链科技在众多网关应用中选择 Apache APISIX 的原因。

- [贡献者，是衡量开源项目的金指标](https://mp.weixin.qq.com/s/G1lQUdZ05_HlZLNcY9tflw)  
秉承 The Apache Way，我们相信，只有活跃的社区才能保证开源项目的未来。同时，我们也希望可以通过更加直观的图表来展示社区活跃度。

Yangxiamao committed:
feat:add img

liuxiran commented on website/events/2021-08-19-weekly-report-2021-8.9-8.15.md:
```diff
@@ -0,0 +1,99 @@
+
+<!--truncate-->
+
+## **导语**
```
```suggestion
## 导语

```

liuxiran commented on website/events/2021-08-19-weekly-report-2021-8.9-8.15.md:
```diff
@@ -0,0 +1,99 @@
+
+Apache APISIX 的[项目官网](https://apisix.apache.org/)和 Github 上的 [issue](https://github.com/apache/apisix/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) 上已经积累了比较丰富的文档教程和使用经验，如果您遇到问题可以翻阅文档，用关键词在 issue 中搜索，也可以参与 issue 上的讨论，提出自己的想法和实践经验。 
+
+## **本周博文推荐**
```
```suggestion
## 本周博文推荐

```

liuxiran commented on website/events/2021-08-19-weekly-report-2021-8.9-8.15.md:
```diff
@@ -0,0 +1,99 @@
+
+## **导语**
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
```
add link for `Apache APISIX`

Yangxiamao replied:
I think it is unnecessary……

liuxiran commented on website/events/2021-08-19-weekly-report-2021-8.9-8.15.md:
```diff
@@ -0,0 +1,99 @@
+
+更新文档 http://apisix.apache.org/docs/apisix/plugins/redirect
+
+## **本周功能特性亮点**
```
```suggestion
## 本周功能特性亮点

```

liuxiran commented on website/events/2021-08-19-weekly-report-2021-8.9-8.15.md:
```diff
@@ -0,0 +1,99 @@
+
+链接：https://github.com/apache/apisix/issues/4800  
+**问题描述**：Admin API 中关于创建 Route 入参 (vars) 的描述有错误
+![](../static/img/event_img/2021-8-19.png)
```
need image  description

Yangxiamao committed:
Update website/events/2021-08-19-weekly-report-2021-8.9-8.15.md

Co-authored-by: liuxiran <liuxiran@apache.org>

Update website/events/2021-08-19-weekly-report-2021-8.9-8.15.md

Co-authored-by: liuxiran <liuxiran@apache.org>

Update website/events/2021-08-19-weekly-report-2021-8.9-8.15.md

Co-authored-by: liuxiran <liuxiran@apache.org>

Update 2021-08-19-weekly-report-2021-8.9-8.15.md

update photo

Update 2021-08-19-weekly-report-2021-8.9-8.15.md

Update 2021-08-19-weekly-report-2021-8.9-8.15.md

Update 2021-08-19-weekly-report-2021-8.9-8.15.md

docs: add the second weekly report

chore: remove descendant of <a> (#478)

feat: add articles directory (#485)

Delete 2021-08-20-weekly-report-2021-8.16-8.22.md

Yangxiamao committed:
docs: add the second weekly report

Yangxiamao committed:
Merge branch 'apache:master' into master

Yangxiamao committed:
Merge branch 'second_weekly_report'
docs: add second weekly report template

juzhiyuan committed:
Rename 2021-08-19-weekly-report-2021-8.9-8.15.md to 2021-08-19-weekly-report.md

juzhiyuan committed:
Rename 2021-08-20-weekly-report-2021-8.16-8.22.md to 2021-08-20-weekly-report.md

juzhiyuan committed:
Update 2021-08-20-weekly-report.md

juzhiyuan committed:
Update 2021-08-19-weekly-report.md

juzhiyuan committed:
Delete 2021-08-20-weekly-report.md

juzhiyuan committed:
Update 2021-08-19-weekly-report.md

juzhiyuan committed:
Update 2021-08-19-weekly-report.md
